packages/storefront/src/components/ComponentMetaOverview.vue

```diff
@@ -2,10 +2,12 @@
   <div>
     <div style="display: flex; justify-content: space-between; margin: 0 0 1rem">
       <div>🚫 = deprecated<br />🛠 = breakpointCustomizable</div>
-      <p-switch @update="toggleProps" :checked="isToggled">Show all prop values</p-switch>
+      <p-switch :theme="$store.getters.platformTheme" @update="toggleProps" :checked="isToggled"
```

Comment:
When is theme retrieved inline from $store like this and when is there are dedicated getter function? Currently it seems to be random mix.

Reply:
I suggest, using it inline when it's only used once per file. As soon as it's used twice then we can abstract it into a getter function. I'll refactor it.